lms/envs/production.py

```diff
@@ -944,3 +944,7 @@ def get_env_setting(setting):
 ########################## Derive Any Derived Settings  #######################
 
 derive_settings(__name__)
+
+############## ADG Specific Settings  #########################
+
+MANDRILL_API_KEY = ENV_TOKENS.get('MANDRILL_API_KEY', None)
```

Comment:
Please move this code above the `derive_settings(__name__)`

Reply:
Can you plz mention a reason why I should move this above?